src/wallet/wallet2.cpp

```diff
@@ -13035,6 +13044,77 @@ void wallet2::throw_on_rpc_response_error(const boost::optional<std::string> &st
   THROW_WALLET_EXCEPTION_IF(*status != CORE_RPC_STATUS_OK, tools::error::wallet_generic_rpc_error, method, m_trusted_daemon ? *status : "daemon error");
 }
 //----------------------------------------------------------------------------------------------------
+
+bool wallet2::save_to_file(const std::string& path_to_file, const std::string& raw, bool is_printable) const
+{
+  if (is_printable || m_export_format == ExportFormat::Raw)
+  {
+    return epee::file_io_utils::save_string_to_file(path_to_file, raw);
+  }
+
+  // Split the long string into multiple lines, similarly to how openssl
+  // prints certificates. And add the magic header.
+  std::string h(tools::base58::encode(raw));
```

Comment:
If you're using base58 now, a number of places should change from "hex". Especially the settings.

Reply:
I don't think the end user cares about the underlying encoding so it would make sense to use something less technical. How about `binary` and `text`?

Reply:
Sure.

Reply:
Maybe ASCII, since text implies, well, text, not seemingly random letters. But whatever you think is appropriate.

Reply:
All right, makes sense. Will update it all, thanks.